react/components/ProductImages/components/Image.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+
+import '../global.css'
+
+/** Image component with 1:1 aspect ratio */
+export default function Image({ src, className, onMouseEnter }) {
```

Comment:
I think it's better to declare the component in a variable, and then `export default Image`, making it easier to add hocs in the future. And arrow functions's usage is encouraged :)

Reply:
Also... this is not actually an `Image` component, right? It renders a div with a background. The name could be more specific about that